packages/eas-cli/graphql-codegen.yml

```diff
@@ -0,0 +1,15 @@
+overwrite: true
+schema: 'http://localhost:3000/--/graphql'
+documents:
+  - 'src/graphql/**/!(*.d).{ts,tsx}'
+generates:
+  src/graphql/generated.tsx:
+    plugins:
+      - 'typescript'
+      - 'typescript-operations'
+    hooks:
+      afterOneFileWrite:
+        - ./annotate-graphql-codegen.sh
+  ./graphql.schema.json:
```

Comment:
If you don't intend on using this JSON file for editor integrations (GraphQL definition autocomplete) then it might make sense to not include this. You'd have one less [pretty gigantic (>440KB)] file to worry about.